### PR TITLE
Edits and 1 question to application deployment doc

### PR DESCRIPTION
**Pull request by sascala**
@marcomonaco I'm assigning this to you because I'm not sure who the correct person should be.

### CODE REVIEW

File: docs/docs/deployments.md [hunk 1]
```diff
@@ -74,55 +72,51 @@ The REST interface allows the force flag for all state-changing operations.
...
+- The new application does not start correctly.
+- The new application does not become healthy.
+- A dependency of the new application was not declared and is not available.
+- The capacity of the cluster is exhausted.
+- The app uses a Docker container and the changes listed at [Running Docker Containers on Marathon]
+(https://mesosphere.github.io/marathon/docs/native-docker.html) were not implemented.
 
-The deployment in this case would take forever.
-To heal the system, a new deployment must be applied to correct the problem with the current deployment.
+To heal the system in cases like these, apply a new deployment to correct the problem with the current deployment.
```
Comment (sascala) [inline docs/docs/deployments.md]: Does the user need to use the --force flag?

File: docs/docs/deployments.md [hunk 2]
```diff
@@ -74,55 +72,51 @@ The REST interface allows the force flag for all state-changing operations.
...
-that state until a future deployment is being made for that app.
+that state until a future deployment is made for that app.
 
-By contrast, the only kind of deployments that can be force-updated safely are those which
-affect single apps only.
-
-Consequently, the only good reason to force a deployment affecting multiple apps is to correct
+The only kind of deployments that can be force-updated safely are those that
+affect single apps only, so the only good reason to force a deployment affecting multiple apps is to correct
 a failed deployment.
```
Comment (sascala) [inline docs/docs/deployments.md]: Code snippet?

File: docs/docs/deployments.md (hunk 1 above)
Comment (aquamatthias) [inline docs/docs/deployments.md]: Yes a force flag is need here.

File: docs/docs/deployments.md (hunk 2 above)
Comment (aquamatthias) [inline docs/docs/deployments.md]: You would post to the apps or groups endpoint to update the failing app.

**Comment by aquamatthias** [comment]
@sascala I close this PR for the moment. Once you have updated the docs, please reopen.

**Pull request closed without merging**